cdap-master/src/main/java/co/cask/cdap/data/tools/SimpleHBaseQueueDebugger.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright © 2015 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.cdap.data.tools;
+
+/**
+ * {@link HBaseQueueDebugger} with simple defaults.
+ */
+public class SimpleHBaseQueueDebugger {
+  public static void main(String[] args) throws Exception {
+    System.setProperty(HBaseQueueDebugger.PROP_SHOW_PROGRESS, "false");
+    System.setProperty(HBaseQueueDebugger.PROP_SHOW_TX_TIMESTAMP_ONLY, "true");
+    HBaseQueueDebugger.main(args);
```

Comment:
Just for completeness sake, can you show SimpleHBaseQueueDebugger's help here when args[1] == "help"? Otherwise, we'll be showing HBaseQueueDebugger's help, right?